Userland/Libraries/LibGfx/Path.h

```diff
@@ -254,14 +254,14 @@ class Path {
     void segmentize_path();
 
     template<PathSegment::Command command, typename... Args>
-    void append_segment(FloatPoint point, Args&&... args)
+    void append_segment(Args&&... args)
     {
-        constexpr auto point_count = sizeof...(Args) + 1;
+        constexpr auto point_count = sizeof...(Args);
         static_assert(point_count == PathSegment::points_per_command(command));
-        m_commands.append(command);
-        // Place the current path point after any extra control points so `m_points.last()` is always the last point.
-        FloatPoint points[] { args..., point };
+        FloatPoint points[] { args... };
+        // Note: This should maintain the invariant that `m_points.last()` is always the last point in the path.
         m_points.append(points, point_count);
+        m_commands.append(command);
```

Comment:
You don't have to do this, but FYI this is the type of thing fold expressions are meant for:
```c++
    template<PathSegment::Command command, typename... Args>
    void append_segment(Args&&... args)
    {
        constexpr auto point_count = sizeof...(Args);
        static_assert(point_count == PathSegment::points_per_command(command));
        // Note: This should maintain the invariant that `m_points.last()` is always the last point in the path.
        (m_points.append(args), ...);
        m_commands.append(command);
    }
```

Reply:
I kinda prefer the current approach, as calling `append()` with the count ensures the capacity for me.

Reply:
Yeah as I was writing that, I was thinking it'd be nice if Vector had a variadic append